src/modules/Interactive.js

```diff
@@ -0,0 +1,14 @@
+const ChangeTaskStatus = (arr, id) => {
+  if (arr[id - 1].completed === true) {
+    arr[id - 1].completed = false;
+  } else {
+    arr[id - 1].completed = true;
+  }
+  return arr;
+};
```

Comment:
- [x] Please consider changing the logic from this function to a simpler one like: 
`const ChangeTaskStatus = (arr, id) => {`
`  arr[id - 1].completed = !arr[id - 1].completed;`
`  return arr;`
 `}`
because you are only changing the state of a boolean variable so you don't actually need the conditional (YAGNI)